edsdme/blocks/announcement-date/announcement-date.js

```diff
@@ -0,0 +1,37 @@
+import { formatDate, getLocale } from '../../scripts/utils.js';
+import { createTag } from '../utils/utils.js';
+
+const CARD_METADATA_PROPERTY_CREATED = 'created';
+
+export default async function init(el) {
+  performance.mark('announcement-date:start');
+
+  let createdDateValue;
+  const cardMetadataEl = document.querySelector('.card-metadata');
+
+  if (cardMetadataEl) {
+    [...cardMetadataEl.children].forEach((row) => {
+      const firstColumn = row.children[0];
+      if (firstColumn && firstColumn.innerText.trim().toLowerCase() === CARD_METADATA_PROPERTY_CREATED) {
+        createdDateValue = row.children[1]?.innerText.trim();
+      }
+    });
+  }
+
+  const createdDate = new Date(createdDateValue);
+  const locale = getLocale();
```

Comment:
@richardhand-fortedigital we can move locale in if statement, to avoid making the call if the statement is false

Reply:
done